### DISCUSSION TITLE
Ghidra console

### DISCUSSION THREAD

**Discussion by chiffreKing** [NationalSecurityAgency/ghidra#3308] [General]
Hi there

I'm working at a script that is printing out a lot of instructions. My problem is that I'm not able to see all the printed lines on the console.
Is the amount of characters that can be printed and shown by the Ghidra console limited?

**Comment by dev747368** [comment]
Yes, its limited by default to 50k.  You can increase this number in the tool options (Edit -> Tool Options -> Console), but be warned, setting this to a higher number can cause Ghidra to bog down when a script is quickly writing a lot of stuff.  An improvement to the console just went into master (https://github.com/NationalSecurityAgency/ghidra/issues/3251), but large scroll back buffers are still an issue.

You should be able to see the same info in your homedir/.ghidra/.ghidra-version/application.log (subject to it being rotated at 50Mb to application.log-date)
